### ISSUE TITLE
Run speed past 30

### ISSUE THREAD

**Issue by DvvCz** [rarsm/rars#106]
In certain cases, your code is a little too slow for comfort with 30 inst/sec but you also don't want it to run unlimited since that could end up running so fast it freezes RARS in some cases... It would be nice if you could set it to some arbitrary number, or the max could be increased a bit.

I was thinking of making a PR for this immediately but it doesn't seem too trivial to do with how it's currently setup, and maybe better to take some time to brainstorm

**Comment by privat** [comment]
Yeah, we could increase the max, but do people really use the feature?

**Comment by DvvCz** [comment]
I gave my use-case above as students may use it, if you don't want to just flat out increase the max, the next best thing would be to implement some way of adjusting it

**Comment by privat** [comment]
I'm OK with incrementing the max. Send a PR :)
Maybe a dialog box that accepts an arbitrary number.
